### ISSUE TITLE
Add all blocks scaffolding blocks

### ISSUE THREAD

**Issue by LouisL4** [rebane2001/mapartcraft#281]
especially moss, it is easy to break and collect

**Comment by ReyvXyd** [comment]
I agree. I would like to try how it will be built with scaffolding. Especially if they are optimized in construction. Please add this because it is necessary to somehow improve the optimization of the construction of artefacts that are built with stairs. Thanks for understanding
